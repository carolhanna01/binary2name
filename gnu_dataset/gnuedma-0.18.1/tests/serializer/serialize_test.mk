#   EDMA: Entorno de Desarrollo Modular y Abierto
#   Object Oriented and Componentware Framework  
#
#   Copyright (C) 1998, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2010
#             by David Mart√≠nez Oliveira
#   
#   This program is free software: you can redistribute it and/or modify
#   it under the terms of the GNU General Public License as published by
#   the Free Software Foundation, either version 3 of the License, or
#   (at your option) any later version.
#
#   This program is distributed in the hope that it will be useful,
#   but WITHOUT ANY WARRANTY; without even the implied warranty of
#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
#   GNU General Public License for more details.
#   
#   You should have received a copy of the GNU General Public License
#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
#

CC=gcc
CFLAGS=`edma-config --cflags-exe`
LIBS=`edma-config --libs-exe`

all: serialize_test
serialize_test: serialize_test.c
	$(CC) $(CFLAGS) $< -o $@ $(LIBS)
configure:
	echo "No configuration is necessary."
clean:
	rm -f core *.o *~ serialize_test
