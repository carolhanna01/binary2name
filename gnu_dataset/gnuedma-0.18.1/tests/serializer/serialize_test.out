
GNU/EDMA 0.17.1 build [Feb  6 2010][Beta][0.17.1] starting up...
EDMA Startup process... OK
(SERIALIZER)-------------------------------------------
(SERIALIZER)Class name : ID
(SERIALIZER)Type = 2 Size = 4
(SERIALIZER)Type = 14 Size = 20736
(SERIALIZER)Marshall done!!! <------------------------->
---------------------[EDMAEnd]----------------------------------
No more process in GNU EDMA. Freeing All Resources...
Cleanup done!!
